packages/core/src/Point.ts

```diff
@@ -7,7 +7,7 @@ import {escape} from './util/escape'
 export interface PointSettings {
   defaultTags?: {[key: string]: string}
   convertTime?: (
-    value: string | number | Date | undefined
+    value: string | number | Date | unknown | undefined
```

Comment:
This reduces to `unknown`:
```suggestion
    value: unknown
```
I don't actually think that is what you want though, it is basically saying, "I'll take anything you give me", which isn't true.  You won't accept an object, or null, or an array, for example.

Reply:
My original aim was to add a bigint to the function signature. I can't, older clients might fail on it. So I have chosen `unknown`, it is indeed not a perfect fit, do you have a better alternative?